config/locales/en_esdl.yml

```
@@ -25,6 +25,14 @@ en:
           More information about Mondaine and ESDL&hellip;
         </a>
       </p>
+    or: 'Or...'
     upload: "Upload an .esdl file"
-    start: "Start your scenario"
+    start: "Start your scenario >>"
```

Personally I'd move the arrow to the button itself, since it isn't part of the translated text:

```suggestion
    start: "Start your scenario"
```

```haml
= f.submit "#{t('esdl.start')} →", class: 'button'
```